packages/styled-components/components/text/index.js

```diff
@@ -6,6 +6,7 @@ import { richText } from '@irvingjs/core/config/html';
 import withThemes from '@irvingjs/styled/components/withThemes';
 import * as defaultStyles from './themes/default';
 import * as htmlStyles from './themes/html';
+import * as responsiveVideoStyles from './themes/responsiveVideo';
```

Comment:
Since it doesn't _have_ to be a video, worth considering maybe "responsive embed" as the name?